_posts/2020-05-13-#17681.md

```diff
@@ -34,7 +34,63 @@ commit:
 
 ## Questions
 
-_Questions to follow soon._
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or
+   NACK?](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)
+   (Don't forget to put your PR review on GitHub.)
+
+2. How does it help users to top up keypools for inactive HD seeds? What is not
+   good about the behavior before this PR?
+
+3. The original motivation for the PR was to remove the restriction that
+   setting a new HD seed wasn't [allowed during Initial Block
+   Download](https://github.com/bitcoin/bitcoin/commit/769b03a83c2aa2b97f344b58dc689be26c6e08e5)?
```

Comment:
It reads like this sentence was intended to be a sentence (the "?" appears to be inadvertent).